Update login token modal and button behavior

The-Dev-Ryan opened the pull request:
Removes orange background from username and password fields in the login token modal, updates modal title to 'Login Token Created', disables the OK button, and changes the Cancel button text to 'Close'. Also added a 'return false' statement to "account_createLoginTokenEx" to keep the modal open upon token creation.

The-Dev-Ryan committed:
Update login token modal and button behavior

Removes orange background from username and password fields in the login token modal,
updates modal title to 'Login Token Created', disables the OK button, and changes
the Cancel button text to 'Close'. Also adds a 'return false' statement to
account_createLoginTokenEx to keep the modal open upon token creation.

si458 commented:
because you have added a string `"Close"` you need to run the translate so its extractes the new string for translations
`cd translate`
`node translate.js` <- let it install missing modules
`node translate.js extractall`
this should really ALWAYS be done when editing the handlebars files
then inspect the `translate.json` file

edit: also `Login Token Created` will need to be translated too
hint: anything with `" "` gets translated so use this for strings to be translated and `' '` for everything else

The-Dev-Ryan committed:
Added Close and Login Token Created

Added Close and Login Token Created

The-Dev-Ryan commented:
> because you have added a string `"Close"` you need to run the translate so its extractes the new string for translations `cd translate` `node translate.js` <- let it install missing modules `node translate.js extractall` this should really ALWAYS be done when editing the handlebars files then inspect the `translate.json` file
> 
> edit: also `Login Token Created` will need to be translated too hint: anything with `" "` gets translated so use this for strings to be translated and `' '` for everything else

Thanks for that mate, I had not looked into the translations side of this :)

The-Dev-Ryan committed:
Add translations for 'Login Token Created'

Added multiple language translations for the 'Login Token Created' message in translate.json to improve localization support.

Tech-Werkz commented:
is this already fixed ? 
I'm still encountering now the Create Token Issues
its not continuing after selection of Token name and Duration
